packages/apputils-extension/src/index.ts

```diff
@@ -615,7 +623,7 @@ namespace Private {
   /**
    * Create a splash element.
    */
-  function createSplash(): HTMLElement {
+  export function createSplash(): HTMLElement {
```

Comment:
You don't use this anywhere, so it probably doesn't need to be exported in this PR, right? Or are you trying to address @Carreau's request here too of exporting some of these things?

Reply:
Yeah, I tried out making my own Nyan splash in the simplest/laziest possible way, by monkeypatching the namespace. But I did not intend to leave that in (for this PR, anyway).

It's gone. I suppose this PR cruft is my justly deserved reward for trying to work on 4 different things at once in the same local source tree.